### PR TITLE
tldr: update page

### PR DESCRIPTION
**Pull request by vanvuvuong**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/tldr.md [hunk 1]
```diff
@@ -27,3 +27,6 @@
 - [l]ist all pages for the current platform and `common`:
 
 `tldr --list`
+
+- [l]ist all available subcommands for the current platform in tldr pages:
+`tldr --list | grep {{command}} | column`
```
Comment (spageektti) [inline pages/common/tldr.md:32]: **
```suggestion
- [l]ist all available subcommands for the current platform in tldr pages:

`tldr --list | grep {{command}} | column`
```
**

File: pages/common/tldr.md [hunk 2]
```diff
@@ -27,3 +27,6 @@
 - [l]ist all pages for the current platform and `common`:
 
 `tldr --list`
+
+- [l]ist all available subcommands for the current platform in tldr pages:
```
Comment (spageektti) [inline pages/common/tldr.md:31]:
```suggestion
- [l]ist all available subcommand pages for a command:
```

**Comment by spageektti** [comment]
Can you please do not force push? Commits are squashed when merging anyway.

File: pages/common/tldr.md (hunk 2 above)
Comment (spageektti) [reply pages/common/tldr.md:31]: Why you resolved this suggestion?

**Comment by vanvuvuong** [comment]
> Can you please do not force push? Commits are squashed when merging anyway.

actually, I was trying to remove duplicate commit

File: pages/common/tldr.md (hunk 2 above)
Comment (vanvuvuong) [reply pages/common/tldr.md:31]: yesterday I skimmed and mistook resolving it

**Review by spageektti — APPROVED** [review]

**Pull request merged**